js/checkout.js

```diff
@@ -0,0 +1,193 @@
+'use strict';
+class Blockonomics {
+
+    constructor({
```

Comment:
The constructor still seems unnecessarily complicated. Please update to the following, unless there is a reason we require the additional overhead
```
constructor({
    checkout_id = 'blockonomics_checkout'
}={}) {
    this.checkout_id = checkout_id
    this.init()
}
```

Reply:
Simplified